depscan/lib/utils.py

```diff
@@ -367,10 +366,3 @@ def get_all_imports(src_dir):
                         import_list.add(pkg)
                         import_list.add(pkg.lower().replace("py", ""))
     return import_list
-
-
-def get_version():
```

Comment:
This function was created to set the version argument in the generated VEX file.

https://github.com/owasp-dep-scan/dep-scan/blob/5724861dda412e98b68c68b56963147aa71e3a17/depscan/cli.py#L406

Any ideas how this might work with the new approach?

Reply:
Fixed this by replacing get_version() by __VERSION__.